### DISCUSSION TITLE
Portabilité des polices ?

### DISCUSSION THREAD

**Discussion by LauwCost** [senshu/Sozi#630] [Q&A]
Bonjour à toutes et tous, 

Ayant constaté des soucis sur certains supports qui n'avaient pas la police que j'avais utilisée, je me suis naturellement posé la question de la "portabilité" des polices utilisées.

Bien sûr, avec un peu d'aide, j'ai résolu le problème en modifiant manuellement l'en-tête mais il semblerait que cette question, si on en croit le [sommaire de la doc ](http://sozi.guide/fr/integrer-les-polices.html) ait été pensée.
Si on m'explique comment procéder et que l'on me donne accès à la page de modification de la doc, je peux contribuer sur ce point si ça aide.

Merci pour les éventuels retours, 
Librement
LauwCost

**Comment by aumouvantsillage** [comment]
> j'ai résolu le problème en modifiant manuellement l'en-tête

Je ne sais pas exactement de quel en-tête il s'agit, mais il y a actuellement deux solutions recommandées :

* Transformer les textes en chemins dans le document SVG
* Inclure les polices dans une feuille de style sous forme de *data URI*

Il y a une discussion en anglais sur ce sujet ici : #595 

> Si on m'explique comment procéder et que l'on me donne accès à la page de modification de la doc

La documentation est hébergée ici : https://github.com/sozi-projects/Sozi-guide
Le système de publication n'est pas habituel et peut demander un effort d'apprentissage.

**Comment by LauwCost** [comment]
Bonjour Guillaume,


Le 01/06/2022 à 22:36, Guillaume Savaton a écrit :
>
>     j'ai résolu le problème en modifiant manuellement l'en-tête
>
> Je ne sais pas exactement de quel en-tête il s'agit,
>
Pardon, je parlais de l'en-tête du fichier html. Mais le soucis 
effectivement exprimé sur certains fils que j'ai lus, c'est les 
révisions de la présentation qui écrasent la modification manuelle.
>
>   * Transformer les textes en chemins dans le document SVG
>
Oui, j'étais tombé sur un fil en anglais après envoi de mon message : il 
y avait quelques limites mineures mentionnées (soulignement des polices, 
...) mais c'est une voie que je m'étais promis de tester et d'essayer 
car elle semblait la plus rapide et efficace devant le point d'après.
>
>   * Inclure les polices dans une feuille de style sous forme de /data URI/
>
> Il y a une discussion en anglais sur ce sujet ici : #595 
> <https://github.com/sozi-projects/Sozi/discussions/595>
>
Ah oui, c'est bien la discussion sur laquelle j'étais tombé. J'avais 
appréhender la solution texte --> chemin mais la suite dépassait mon 
anglais et mes connaissances techniques.
>
>     Si on m'explique comment procéder et que l'on me donne accès à la
>     page de modification de la doc
>
> La documentation est hébergée ici : 
> https://github.com/sozi-projects/Sozi-guide
> Le système de publication n'est pas habituel et peut demander un 
> effort d'apprentissage.
>
Je vais regarder et voir aussi avec la personne qui m'a aidé hier.

Merci encore pour cette belle solution libre !

Librement
